src/flex/FlexView.js

```diff
@@ -68,6 +68,16 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    if (this.props.basis === 'auto') {
+      warn('Default basis is auto. Force it to auto will leave shrink true as default');
```

Comment:
"Force it to auto" is an error in english :stuck_out_tongue_closed_eyes: 
anyway, I would change the first part to `basis is "auto" by default: forcing it to "auto" ....`
(we're already using double quotes for props in other warnings I think)